packages/php/remote-inbox-notification-testing/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "woocommerce/remote-inbox-notification-testing",
+	"description": "Remote inbox notification testing suite",
+	"type": "library",
+	"license": "GPL-3.0-or-later",
+	"autoload": {
+		"psr-4": {
+			"Woocommerce\\WoocommerceRemoteInboxNotificationTesting\\": "src/"
```

Comment:
This is an optional nitpick. The namspace that we have generally standardized on is `Automattic\WooCommerce\Tests\*`. If there isn't any code for this yet, could we change this to something like `Automattic\WooCommerce\Tests\RemoteInboxNotifications`.

```
# git grep RemoteInboxNotification -- '*.php' | grep namespace | cut -f2 -d: | sort -u
namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;
namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications\Transformers;
```